#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise NameError exceptions with a Message"""
    raise NameError(message)
